front-end-tools/src/reading_from_blockchain.ts

```diff
@@ -0,0 +1,5 @@
+import { JsonRpcClient } from '@concordium/web-sdk';
+
+export async function accountInfo(rpcClient: JsonRpcClient, account: string) {
```

Comment:
COuld we please not use the jsonrpcclient seeing it's deprecated?

is there something preventing that?